Dockerfile

```diff
@@ -1,22 +1,32 @@
-FROM golang:1.11.4-alpine3.8 AS builder
-LABEL maintainer="joona@kuori.org"
+FROM golang:1.12.0-alpine3.9 AS builder
```

Comment:
`golang:1-alpine3.9` would be better if we don't need to fix the go version.

Reply:
I don't like the approach to just have the major version fixed. If you checkout an older version tag of this project and you want to build the docker-image out of it, there is then no simple way to findout the used golang version. Maybe it was 1.11.x, but it can be 1.12.x.
We can freeze the golang version to the minor version if thats ok for you. like `golang:1.12-alpine3.9`

Reply:
That's fine with me, but since you went from `1.11` to `1.12` it really doesn't seem like it matters. Does the built binary not show you which golang version was used?